request.go

```diff
@@ -186,8 +186,8 @@ func (a *Request) Sanitize(allowedParameters []string) Requester {
 	b.ID = a.GetID()
 	b.Form = url.Values{}
 	for k := range a.Form {
```

Comment:
Typically only the first value is set in `url.Values` but this is still a good merge. It would be nice though to have a test case that covers the fixed behavior to prevent regression.